controllers/logs.js

```diff
@@ -1,77 +1,83 @@
 const express = require('express');
-
-const url = require('url');
-const crypto = require('crypto');
-const ipaddr = require('ipaddr.js');
-
-const api = require('../api');
-const helper = require('../helpers/googleAnalytics');
-
+const request = require('request');
 const logger = require('../helpers/logger');
 
-logger.info('Google Analytics Tracking ID: ' + process.env.SW_GOOGLE_ANALYTICS_TRACKING_ID);
-
 const router = express.Router();
 
 router.use(require('../helpers/authentication').authChecker);
```

Comment:
this meqans we are not able to check loggedout stuff, can we allow valid csrf instead?

Reply:
Currently we only check logged in stuff